aurora/grids_utils.py

```diff
@@ -743,17 +747,16 @@ def build_aug_geqdsk(shot, time):
     geqdsk['fluxSurfaces']['Z0'] = Z0
     geo = geqdsk['fluxSurfaces']['geo'] = {}
     
-    pf = equ.get_profile('PFL')[it]
```

Comment:
One small caveat CLISTE is using a different definition of poloidal flux compared to EFIT, it differs by a factor of 2pi. I think that it does not matter here, but if you need to calculate anything which depends on the absolute value of Psi, you should be aware of it.

Reply:
thanks for mentioning this, @odstrcilt . Lots of little things to be careful about... 

This should be good enough as a first implementation, although this can likely be improved with extra help from @tardini over time.